benchmarks/pairwise_dist.py

```diff
@@ -6,7 +6,8 @@
 
 
 class Benchmark(BaseBenchmark):
-    def __init__(self, functions, datasets):
+    def __init__(self, name, functions, datasets):
+        self.name = name
```

Comment:
For OO design consistency, calling `BaseBenchmark`'s constructor is better:
```suggestion
        super().__init__(name)
```